.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
+install: bundle install --retry=3 --deployment --without development production
 cache: bundler
-sudo: true
+sudo: false
```

Comment:
I am not sure what te default behavior is. But the entire file is copied from Sufia.